backend/oasst_backend/utils/hugging_face.py

```diff
@@ -4,14 +4,20 @@
 import aiohttp
 from loguru import logger
 from oasst_backend.config import settings
+from oasst_backend.models import Message
+from oasst_backend.prompt_repository import PromptRepository
 from oasst_shared.exceptions import OasstError, OasstErrorCode
 
 
 class HfUrl(str, Enum):
-    HUGGINGFACE_TOXIC_ROBERTA = ("https://api-inference.huggingface.co/models/unitary/multilingual-toxic-xlm-roberta",)
+    HUGGINGFACE_TOXIC_ROBERTA = "https://api-inference.huggingface.co/models/unitary/multilingual-toxic-xlm-roberta"
     HUGGINGFACE_FEATURE_EXTRACTION = "https://api-inference.huggingface.co/pipeline/feature-extraction"
 
 
+class HfModel(str, Enum):
```

Comment:
In analogy to below, this should probably be called something like `HfClassificationModel`

Reply:
Done thanks!